pkg/cli/root.go

```diff
@@ -771,6 +776,42 @@ func LoadConfiguration(config *config.Config, configPath string) error {
 	return nil
 }
 
+func updateLDAPConfig(conf *config.Config) error {
+	if conf.HTTP.Auth == nil || conf.HTTP.Auth.LDAP == nil {
+		return nil
+	}
+
+	if conf.HTTP.Auth.LDAP.CredentialsFile == "" {
```

Comment:
This credentials file may be empty (or not specified in config)
But the rest is still valid. We could have anonymous LDAP binds?

Reply:
Why would the current implementation need changes? It doesn't make an update to the credentials in the config if the file is not specified?

Reply:
Similar to sync credentials or all other credentials

Reply:
CredentialsFile can be nil and add a test case to test LDAP code works after that.